docs/catalogue/cat_network-data-manager.md

```diff
@@ -176,26 +160,18 @@ Subcohorts_ sheet. |
 <sup>*Table 6. Description of the columns that can be filled out for CollectionEvents. \* = mandatory;* *\*\** *
 contact* [*molgenis-support@umcg.nl*](mailto:molgenis-support@umcg.nl) *to add AgeCategories*</sup>
 
-![Figure 7. Example of CollectionEvents filled out in Excel](../img/cat_target-collection-events-sheet.png ':size=500')
-
-<sup>*Figure 7. Example of CollectionEvents filled out in Excel.*</sup>
-
 #### *Subcohorts* sheet
 
```

Comment:
Subcohorts sheet contain more columns then listed below: ageGroups, disease, inclusionCriteria, countries, regions, supplementaryInformation

Reply:
these can be filled via forms (see section on cohort rich metadata) and not via the templates, since they contain a lot of look up lists and are easier to fill via forms.